examples/hosts/app-integration/external-data/src/model-interface/index.ts

```diff
@@ -97,6 +97,11 @@ export interface ITaskList extends IEventProvider<ITaskListEvents> {
      */
     readonly deleteTask: (id: string) => void;
 
+    /**
+     * Get the sync status of the fluid container. True if synced.
```

Comment:
It isn't totally clear to me what this state represents. Does this means all of our local changes have been sent to the external data service? Or that we have reflected all known changes _from_ the external data service? Or both?

Reply:
In my understanding, this state represents whether our local changes are synced with everyone else's in Fluid. If this was false, that would mean we need to sync & or perform some conflict resolution.

Reply:
We shouldn't need to track fluid data sync'ing across clients (we can trust fluid to do this correctly).  We just need to track whether there is discrepancy between the fluid data and the saved data.  I think there's a lot of overlap between what you're looking at and what @ghoshkaj is looking at - have y'all chatted about how you're going to reconcile the two?